specs/latest/2.0/index.html

```diff
@@ -699,12 +694,23 @@ <h2 class="no-toc">Table of contents</h2>
   const GLenum MAX_CLIENT_WAIT_TIMEOUT_WEBGL                 = 0x9247;
 
   /* Buffer objects */
+  void bufferData(GLenum target, BufferDataSource? srcData, GLenum usage,
+                  optional unsigned long srcOffset = 0,
+                  optional unsigned long length = 0);
+  void bufferSubData(GLenum target, GLintptr dstByteOffset,
+                     BufferDataSource srcData,
+                     optional unsigned long srcOffset = 0,
+                     optional unsigned long length = 0);
+
```

Comment:
I think it should be "BufferDataSource?" to be backward compatible. If it's null and srcOffset or length are not zero, we should generate INVALID_VALUE.